app/core/tests/test_models.py

```diff
@@ -38,14 +38,14 @@ def test_container(scoring_container):
     assert scoring_container.uri == "ghcr.io/robbason/score-coords"
 
 
-def test_file_value(input_elements, molfile_type):
+def test_file_value(input_elements, molfile_type):  # pylint: disable=unused-argument
```

Comment:
esp since pylint often needs these arguments to know which fixtures to instantiate

Reply:
This doesn't tell pylint to ignore line line, it will still do the dynamic analysis it does, it just won't throw the error from the unused-argument 

But I will have pylint skip the tests.